guide/interactions/modals.md

```diff
@@ -0,0 +1,117 @@
+# Modals
+
+With modal interactions you can create pop-out forms that allow users to provide you with formatted input. We'll cover how to create, send and receive modals using discord.js!
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) page](/interactions/slash-commands.md). Please carefully read those first so that you can understand the methods used in this section.
+:::
```

Comment:
This page is still in version 13 (not the problem here), so shouldn't the check `if (!interaction.isCommand()) return;` be `if (!interaction.isChatInputCommand()) return;`? The ping command is a chat input command.